templates/patroni.yml.j2

```diff
@@ -99,6 +99,7 @@ postgresql:
   pgpass: /tmp/pgpass
   pg_hba:
     - local all backup peer map=operator
+    - local all operator password
```

Comment:
This is plaintext password auth. It should be fine for socket connections and in line with how monitoring connects. Alternatively we can demand md5 or scram.